build.py

```diff
@@ -0,0 +1,10 @@
+from distutils.extension import Extension
+
+from Cython.Build import cythonize
+
+# empty extension module so build machinery recognizes package as platform-specific
+extensions = [Extension("httpstan.empty", sources=["httpstan/empty.pyx"])]
```

Comment:
Will there be a `httpstan.empty`?

Should we go with `_empty`

Reply:
It's not really a private module.

I think we're going to be stuck with this. It would be nice to translate empty.pyx into plain C so we didn't need to worry about installing Cython. One could also try to use pybind11.

Reply:
I think there is possibility to also create Extension without sources, but I think it might still need some compiled file, so calling it wouldn't crash.